src/commands/upload_file.ts

```diff
@@ -121,6 +142,32 @@ function getSingleFile(parameters: ParametersHelper, parentFolderId: FolderID):
 	return [singleParameter];
 }
 
+async function getCustomMetaDataWithIpfsCid({
```

Comment:
Is it possible to handle this in the parameter helper?

Reply:
Following up on this, perhaps `getCustomMetaData` would check for the `--ipfs-add` flag internally

Reply:
Moves the function to parameters helper: 3167dbc2ae8c54570ce919a11fd028dc53dce5f9
And adds unit tests: c703ac2ffb3f32cfe1be4c2a48a63a15b2fd6b8c